extensions/reactive-datasource/runtime/src/main/java/io/quarkus/reactive/datasource/runtime/DataSourceReactiveRuntimeConfig.java

```diff
@@ -145,4 +146,10 @@ public class DataSourceReactiveRuntimeConfig {
      */
     @ConfigItem
     public Optional<String> name;
+
+    /**
+     * Additional properties for the client, which will be sent to server at the connection start.
+     */
+    @ConfigItem
+    public Map<String, String> properties;
```

Comment:
Perhaps, for consistency, we could call this `additionalProperties`:

https://github.com/quarkusio/quarkus/blob/15677f05c40f6257922e55610321baadfe9c8a9c/extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSourceJdbcRuntimeConfig.java#L135-L139

Reply:
Sure I can do that. I was staying consistent with the `getProperties`/`setProperties`/`addProperty` on `io.vertx.sqlclient.SqlConnectOptions`

Reply:
Thank you